src/Domain/CoordinateMaps/Frustum.cpp

```diff
@@ -22,10 +22,13 @@
 
 namespace domain {
 namespace CoordinateMaps {
+
 Frustum::Frustum(const std::array<std::array<double, 2>, 4>& face_vertices,
                  const double lower_bound, const double upper_bound,
                  OrientationMap<3> orientation_of_frustum,
-                 const bool with_equiangular_map) noexcept
+                 const bool with_equiangular_map,
+                 const double projective_scale_factor,
+                 const bool use_suggested_value) noexcept
```

Comment:
Could use better option name that answers the question "suggested value of what"?  How about `auto_projective_scale_factor` or `use_auto_projective_scale_factor`?